modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavascriptClientCodegen.java

```diff
@@ -1,35 +1,19 @@
 package io.swagger.codegen.languages;
 
 import com.google.common.base.Strings;
-
-import io.swagger.codegen.CliOption;
-import io.swagger.codegen.CodegenConfig;
-import io.swagger.codegen.CodegenConstants;
-import io.swagger.codegen.CodegenModel;
-import io.swagger.codegen.CodegenOperation;
-import io.swagger.codegen.CodegenProperty;
-import io.swagger.codegen.CodegenType;
-import io.swagger.codegen.DefaultCodegen;
-import io.swagger.codegen.SupportingFile;
+import io.swagger.codegen.*;
```

Comment:
We prefer not using wildcard import: https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

Ref: https://github.com/swagger-api/swagger-codegen/blob/master/CONTRIBUTING.md#style-guide